app/controllers/maintenance_tasks/application_controller.rb

```diff
@@ -5,18 +5,9 @@ module MaintenanceTasks
   #
   # Can be extended to add different authentication and authorization code.
   class ApplicationController < MaintenanceTasks.parent_controller.constantize
-    BULMA_CDN = "https://cdn.jsdelivr.net"
-
     content_security_policy do |policy|
```

Comment:
That didn't work:

```
Expected [#<Selenium::WebDriver::LogEntry:0x00007f91196cab18 @level="SEVERE", @timestamp=1716932895557, @message="http://127.0.0.1:34633/maintenance_tasks - Refused to load the script 'http://127.0.0.1:34633/assets/maintenance_tasks-40eb5c80423e97745265599b46535a620b56d705e581cc427891b5460b2f7ccf.js' because it violates the following Content Security Policy directive: \"script-src 'none'\". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.\n">, #<Selenium::WebDriver::LogEntry:0x00007f91196caaf0 @level="SEVERE", @timestamp=1716932895559, @message="http://127.0.0.1:34633/maintenance_tasks - Refused to load the script 'http://127.0.0.1:34633/assets/maintenance_tasks-40eb5c80423e97745265599b46535a620b56d705e581cc427891b5460b2f7ccf.js' because it violates the following Content Security Policy directive: \"script-src 'none'\". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.\n">] to be empty.

```

Reply:
👍🏽, thanks for investigating